product-schema.json

```diff
@@ -0,0 +1,502 @@
+{   "$schema" : "http://json-schema.org/draft-07/schema#",
+	"$id": "",
+	"$comment" : "Digital Public Good Metaschema: JSON Schema",
+	"title": "Digital Public Good",
+ 	"type": "object",
+  "required": [ "name", "description", "license", "repositoryURL", "SDGs", "type"],
```

Comment:
@dial-registry: In my experience, there are data and standard products that do not have repos, so we cannot make `repositoryURL` a required field. 

On the other hand, I would make `organization` required because there will always be someone or a group of someones behind a product in some form or another, so I think it is appropriate to make orgs a required field.

Reply:
Agreed. I will push that change.

Reply:
Fixed in 77be7033a9580356bcd70300856773fd055cc352. Thanks